ui/component/or-asset-viewer/src/index.ts

```diff
@@ -973,9 +972,9 @@ async function getLinkedUserInfo(userAssetLink: UserAssetLink): Promise<UserAsse
     const userId = userAssetLink.id!.userId!;
     const username = userAssetLink.userFullName!;
 
-    const roleNames = await manager.rest.api.UserResource.getUserRoles(manager.displayRealm, userId)
+    const roleNames = await manager.rest.api.UserResource.getUserClientRoles(manager.displayRealm, userId, OPENREMOTE_CLIENT_ID)
```

Comment:
Shouldn't we use `managerConfig.clientId` instead of `OPENREMOTE_CLIENT_ID` here as well?